draft-kuhn-quic-bdpframe-extension-01.xml

```diff
@@ -273,6 +286,12 @@ BDP Frame {
                 	A BDP Frame contains the following fields: 
                 	<list style="symbols">
 			<t>
+			Hash (secured_hash): 
+			The secured_hash is generated by the sender using other 
+			parameters of the BDP Frame. The sender encrypts the hash so 
+			that the client can not read it. 
```

Comment:
/BDP Frame/BDP_Frame/ which is correct?

Reply:
Fixed in commit https://github.com/NicoKos/resume_connection/commit/b6cf8df1573405b060e3cfcb62b1e0efcb38146d